pkg/executor/join/hash_join_stats.go

```diff
@@ -202,6 +225,17 @@ func (e *hashJoinRuntimeStatsV2) String() string {
 		}
 		buf.WriteString("}")
 	}
+	if e.spill.round > 0 {
```

Comment:
Is it possible that e.spill.round is very large? If so, I don't think all the data should be printed. Instead, some aggregated info may be more readable.

Reply:
> Is it possible that e.spill.round is very large? If so, I don't think all the data should be printed. Instead, some aggregated info may be more readable.

It will not be very large.